docs/releaseimageverify.md

```diff
@@ -0,0 +1,75 @@
+# Release Image Verification
+
+## Overview
+
+Currently during upgrades, OpenShift 4 checks that the release image
+that is being requested in upgrade is trusted. By default it only trusts
+release images that are published by Red Hat. This is very important
+since release image is the definition of what runs in the cluster and
+therefore must be verified before it is used.
+
+Similarly, currently Hive uses the release image specified for the
+cluster to extract binaries like oc, openshift-install and then executes
+them in internal environment (Hive clusters), it is important that Hive
+enforce that the content is trusted before running it.
+The mechanism that is used by OpenShift during upgrades should be the
+best way to establish and enforce that trust.
+
+## How verification works?
+
+for details see https://github.com/openshift/cluster-update-keys
+
+The definition of trust is defined in an ConfigMap data keys,
+
+store-*:
+These define the ways in which signature(a) for specific release image
+digest can be fetched.
+Multiple stores can be used to provide resilient source of signatures
+
+verifier-public-key-*:
+This is the public key that will be used to verify the signature and
+release image are indeed trusted.
+Multiple keys imply that the digest and signatures must be trusted by
+every key.
+
+OpenShift publishes this config map by default in each cluster at
+openshift-config-managed/release-verification . So if Hive is running on
+OpenSHift 4 and it intends to allow only release images published by Red
+Hat, it can use this config map as definitions of trust.
+
+The verification requires digest of the release image, the
+repository or source of registry is not used for verification. Therefore
+release images with tag cannot be verified using this method.
```

Comment:
Looks like OSD is already using digests so that's a win.

@jnpacker could you confirm if ACM is populating ClusterImageSets with tags or digests?